### ISSUE TITLE
Omeka S to-do: address alt text issue with default thumbnail

### ISSUE THREAD

**Issue by erinchambers** [CDRH/accessibility#2]
The default thumbnail (that Omeka S uses when an item has no media) does not have any alt text. It MUST have alt text because it is linked, and the text link for that item is in a separate link. Thus, empty alt (`alt=""`) with an ARIA role (` role="presentation"`) is not a valid solution in this case.

![Image](https://github.com/user-attachments/assets/09bcad93-292b-4cd4-a8da-deb13ada4ad1)

**Comment by erinchambers** [comment]
I'm going to wrap this in with the ADA Title II Project - moving it there instead of Bug Squashing 2025.

**Comment by erinchambers** [comment]
`alt="File icon, links to ____"` maybe

**Comment by erinchambers** [comment]
Fixed in an  official Omeka S release
